pkgs/by-name/im/immich-machine-learning/package.nix

```diff
@@ -1,46 +1,43 @@
 {
   lib,
-  src,
   fetchFromGitHub,
   immich,
   python3,
-  # Override Python packages using
-  # self: super: { pkg = super.pkg.overridePythonAttrs (oldAttrs: { ... }); }
-  # Applied after defaultOverrides
-  packageOverrides ? self: super: { },
 }:
 let
-  defaultOverrides = self: super: {
-    pydantic = super.pydantic_1;
+  python = python3.override {
+    self = python;
 
-    versioningit = super.versioningit.overridePythonAttrs (_: {
-      doCheck = false;
-    });
+    packageOverrides = self: super: {
+      pydantic = super.pydantic_1;
 
-    albumentations = super.albumentations.overridePythonAttrs (_: rec {
-      version = "1.4.3";
-      src = fetchFromGitHub {
-        owner = "albumentations-team";
-        repo = "albumentations";
-        rev = version;
-        hash = "sha256-JIBwjYaUP4Sc1bVM/zlj45cz9OWpb/LOBsIqk1m+sQA=";
-      };
-    });
-  };
+      versioningit = super.versioningit.overridePythonAttrs (_: {
+        doCheck = false;
+      });
 
-  python = python3.override {
-    self = python;
-    packageOverrides = lib.composeExtensions defaultOverrides packageOverrides;
+      albumentations = super.albumentations.overridePythonAttrs (_: rec {
+        version = "1.4.3";
+        src = fetchFromGitHub {
+          owner = "albumentations-team";
+          repo = "albumentations";
+          rev = version;
+          hash = "sha256-JIBwjYaUP4Sc1bVM/zlj45cz9OWpb/LOBsIqk1m+sQA=";
+        };
+      });
+    };
   };
 in
 python.pkgs.buildPythonApplication {
   pname = "immich-machine-learning";
   inherit (immich) version;
-  src = "${src}/machine-learning";
+  src = "${immich.src}/machine-learning";
   pyproject = true;
 
   postPatch = ''
     substituteInPlace pyproject.toml --replace-fail 'fastapi-slim' 'fastapi'
+
+    # AttributeError: module 'cv2' has no attribute 'Mat'
```

Comment:
To me this smells like some issue with the way we package opencv4.
cc @philiptaron @SuperSandro2000 @mweinelt